recipes/uvw/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "uvw" %}
+{% set version = "v2.12.1_libuv_v1.44" %}
+{% set sha256 = "3460842778e91e7d2fae4201e49e4521e9d94fbbf3891ae6c52d3c9fc0673598" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
```

Comment:
The version suffix seems to track which (minimal?) version of libuv gets vendored into uvw. This versioning scheme likely does not play nice with conda. Could it just be `2.12.1`?

Reply:
Yes, it can be `2.12.1` but we still have to save the currect compatible `libuv` version, so I added it with another variable and constructed the full filename with both `uvw` and `libuv` version variables.